test/jdk/java/awt/TrayIcon/DisposeInActionEventTest/DisposeInActionEventTest.java

```diff
@@ -100,30 +93,30 @@ public void start() {
         g.fillRect(6, 6, 20, 20);
         g.dispose();
 
-        systemTray = SystemTray.getSystemTray();
-        trayIcon = new TrayIcon(img);
+        SystemTray systemTray = SystemTray.getSystemTray();
+        TrayIcon trayIcon = new TrayIcon(img);
         trayIcon.setImageAutoSize(true);
         trayIcon.addActionListener(ev -> {
-            Sysout.println(ev.toString());
+            textArea.append(ev.toString() + "\n");
             systemTray.remove(trayIcon);
             new Thread(() -> {
                 try {
                     Thread.sleep(1000);
                     systemTray.add(trayIcon);
                 } catch (AWTException | InterruptedException e) {
-                    Sysout.println(e.toString());
-                    Sysout.println("!!! The test coudn't be performed !!!");
+                    textArea.append(e.toString() + "\n");
+                    textArea.append("!!! The test couldn't be performed !!!\\n");
                 }
```

Comment:
Fail the test?

Print the stack trace of the exception with `e.printStackTrace()` — it would help analysing the failure.